app/controllers/concerns/label_helper.rb

```diff
@@ -0,0 +1,10 @@
+module LabelHelper
```

Comment:
`XYCHelper` naming convention is (mostly) used in Rails views to share reusable code. We could evaluate a different naming convention for controller concerns.